app/packages/core/src/components/ColorModal/JSONViewer.tsx

```diff
@@ -2,20 +2,28 @@ import { useTheme } from "@fiftyone/components";
 import { isValidColor } from "@fiftyone/looker/src/overlays/util";
 import * as fos from "@fiftyone/state";
 import Editor from "@monaco-editor/react";
-import React, { useEffect, useRef, useState } from "react";
+import React, { useEffect, useMemo, useRef, useState } from "react";
 import { useRecoilValue } from "recoil";
 import { COLOR_SCHEME } from "../../utils/links";
 import { ActionOption } from "../Actions/Common";
 import { Button } from "../utils";
 import { SectionWrapper } from "./ShareStyledDiv";
 import { validateJSONSetting } from "./utils";
 
-const JSONViewer: React.FC = ({}) => {
+const JSONViewer: React.FC = () => {
   const themeMode = useRecoilValue(fos.theme);
   const theme = useTheme();
   const editorRef = useRef(null);
-  const setting = useRecoilValue(fos.sessionColorScheme);
-  const { setColorScheme } = fos.useSessionColorScheme();
+  const sessionColor = useRecoilValue(fos.sessionColorScheme);
+  const setting = useMemo(() => {
+    return {
```

Comment:
could be safer and makes sure `sessionColor` is defined before accessing its properties